KutokAccounting/Platforms/Windows/App.xaml.cs

```diff
@@ -19,5 +21,10 @@ public App()
 		this.InitializeComponent();
 	}
 
-	protected override MauiApp CreateMauiApp() => MauiProgram.CreateMauiApp();
+	protected override MauiApp CreateMauiApp()
+	{
+		var app = MauiProgram.CreateMauiApp();
```

Comment:
Можна не створювати змінну.
```suggestion
		return MauiProgram.CreateMauiApp();
```